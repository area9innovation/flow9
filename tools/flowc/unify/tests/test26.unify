import maybe;

.(.value : α10001)  grow_right {None<α10000> .. *};

