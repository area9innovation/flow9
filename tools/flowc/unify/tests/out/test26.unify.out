Expected .value in None
Expected .value in None
